v2_ex46.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import csv
+from collections import OrderedDict
+
+filename = 'text_file.csv'
+
+with open(filename, newline ='') as csvfile:
+    linereader = csv.reader(csvfile, delimiter=',')
+    string_of_words = ''
+    for rows in linereader:
+        for row in rows:
+            string_of_words += row + ' '
+            list_of_words = string_of_words.split()
+
+    unique_words = set(list_of_words)
+
+    frequency_tracker = {}
+    for word in unique_words:
+        if word in list_of_words:
```

Comment:
I'm actually not sure if you need line 26. Since you created `unique_words` from `list_of_words`, you know for sure that `word` is in the `list_of_words`. Second, I think you can just iterate through the `unique_words` set and then count each word in the `list_of_words`:

```
frequency_tracker = {}
for word in unique_words:
  frequency = list_of_words.count(word)
  frequency_tracker[word] = frequency
```

That eliminates some unneeded work. I like your approach of using `count` instead of iterating through `list_of_words` and incrementing a dictionary count `+= 1` each time you encounter a word (which is the approach most people take in this situation).